ftva_lab_data/views.py

```diff
@@ -57,3 +62,54 @@ def view_item(request, item_id):
     item = SheetImport.objects.get(id=item_id)
 
     return render(request, "view_item.html", {"item": item})
+
+
+def view_items(request: HttpRequest) -> HttpResponse:
```

Comment:
This view should be renamed to better show its purpose, corresponding to the template rename.